src/utils/hitConverter.js

```diff
@@ -8,13 +8,13 @@ const unflatten = require('flat').unflatten;
 
 class HitConverter{
 
-    constructor(customSymbolizers = []){
+    constructor(customSymbolizer = undefined){
         // TODO: Keep a dictionary of mapping info here to speed up future queries.
 
-        this.customSymbolizers = customSymbolizers;
+        this.customSymbolizer = customSymbolizer;
     }
 
-    featureFromHit(hit, indexConfig, mapping=undefined) {
+    featureFromHit(hit, indexConfig, mapping=undefined, maxAllowableOffset = 1) {
```

Comment:
methods/functions with more than two parameters should be converted to a single object parameter:

```
featureFromHit({ hit, indexConfig, mapping=undefined, maxAllowableOffset = 1 })
```

unless for some reason this would cause a breaking change.

Reply:
I don't think it would hurt anything. Is this a koop convention?

Reply:
A general coding convention we try to follow - taken from Clean Code by Robert Martin.  https://medium.com/@huytrongnguyen1985/lessons-learnt-from-the-clean-code-robert-c-martin-cecbe2b09139.  Less applicable to javascript, but still helps.

Reply:
I updated this function to take hit and indexConfig as they are required and then an options object with the other arguments. Let me know if that works.